src/locale.js

```diff
@@ -1,13 +1,19 @@
+import { merge } from 'lodash/fp'
```

Comment:
Yeah, that's what I wanted, to merge each namespace inside the locale object.
![image](https://user-images.githubusercontent.com/1283200/28337450-0cf56652-6bdc-11e7-81b5-3587d747b700.png)

But I think you are right, I'm going to just make an or case instead of merging. (`export let string = customLocale.string || {...}`).